src/Entity/SolrFieldType.php

```diff
@@ -60,42 +60,42 @@ class SolrFieldType extends AbstractSolrEntity implements SolrFieldTypeInterface
    *
    * @var array
    */
-  protected $field_type;
+  protected $fieldType;
```

Comment:
I'm not sure if that change would break YAML file parsing. At least it did in earlier Drupal version.